web/src/app/util/RequireConfig.js

```diff
@@ -126,14 +128,18 @@ export default function RequireConfig(props) {
   if (wantIsAdmin && !isAdmin) {
     return elseValue
   }
-  if (configID && !test(config[configID])) {
+
+  const cfgOpt = config.find((c) => c.id === configID)
+  if (configID && !test(cfgOpt.value)) {
     return elseValue
   }
 
+  if (!children) return null
   return React.Children.map(children, (child) =>
     React.cloneElement(child, _.omit(rest, Object.keys(child.props))),
   )
 }
+
```

Comment:
The falsy check is redundant: `React.Children.map` already handles this internally.

https://reactjs.org/docs/react-api.html#reactchildrenmap
```suggestion
  return React.Children.map(children, (child) =>
    React.cloneElement(child, _.omit(rest, Object.keys(child.props))),
  )
}
```

Reply:
That makes sense, but I was getting an error when children wasn't given to the component because children was undefined or something to that extent, iirc (the proptypes for `RequireConfig` do not have children as required)

Reply:
The prop type may need to be adjusted -- otherwise there's a PR (though stale) to convert RequireConfig to typescript which should resolve any remaining type complaints in the console when it gets merged.

Reply:
Addressed by using `<React.Fragment>{children}</React.Fragment>` as in the `convert to typescript` PR for this file. Seems like a sound approach, since the parent will have access to all props, there's no point to pass the props to `RequireConfig` and have it spread when it clones if the props can be passed to that child much more transparently from the origin component.

ie:

```
const { foo, bar } = props

return (
  <RequireConfig>
    <SomeComponent foo={foo} bar={bar} />
  </RequireConfig>
)
```